### PR TITLE
create time show

### PR DESCRIPTION
**Pull request by thucpv-1748**
https://edu-redmine.sun-asterisk.vn/issues/9647

### CODE REVIEW

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    /**
+     * ListShowFilmController constructor.
+     * @param TimeShowInterface $model
+     * @param FilmInterface $listfilm
+     * @param RoomInterface $room
+     */
+    public function __construct(
+        TimeShowInterface $model,
+        FilmInterface $listfilm,
+        RoomInterface $room
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 39</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        $this->model = $model;
+        $this->listfilm = $listfilm;
+        $this->room = $room;
+    }
+
+    /**
+     * @param Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function createListShowFilm(Request $request)
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 51</br>Expected 1 space(s) after TRY keyword; 0 found

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    {
+        try{
+            $this->model->create($request->all());
+
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 55</br>Expected 1 space after closing brace; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 55</br>Expected 1 space(s) after closing parenthesis; found 0

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        }catch (\Exception $e){
+            return redirect()->back()->with('error', $e->getMessage());
+        }
+    }
+
+    /**
+     * @param Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function updateListShowFilm(Request $request)
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 66</br>Expected 1 space(s) after TRY keyword; 0 found

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+    {
+        try{
+            $this->model->update($request->id, $request->all());
+
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 70</br>Expected 1 space after closing brace; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 70</br>Expected 1 space(s) after closing parenthesis; found 0

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+        return view('layout.backend.listshowform', compact('rooms', 'listfilms'));
+    }
+
+
+    /**
+     * @param $id
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\Http\RedirectResponse|\Illuminate\View\View
+     */
+
+    public function editTimeShow($id)
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 95</br>Line indented incorrectly; expected at least 8 spaces, found 7

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+    {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 96</br>Line indented incorrectly; expected 8 spaces, found 7

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+       $model = $this->model->findOrFail($id);
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 97</br>Line indented incorrectly; expected at least 12 spaces, found 11

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+       if ($model) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 98</br>Line indented incorrectly; expected at least 12 spaces, found 11

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+           $timeshow = $model;
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 99</br>Line indented incorrectly; expected at least 12 spaces, found 11

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+           $rooms = $this->room->all();
+           $listfilms = $this->listfilm->all();
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 101</br>Line indented incorrectly; expected at least 12 spaces, found 11

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 102</br>Line indented incorrectly; expected 8 spaces, found 7

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+           return view('layout.backend.listshowform', compact('timeshow', 'rooms', 'listfilms'));
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 103</br>Line indented incorrectly; expected at least 12 spaces, found 11

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+       } else {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 104</br>Line indented incorrectly; expected 8 spaces, found 7

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+           return redirect()->back()->with('error', __('not found!'));
+       }
+    }
+
+    /**
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+
+
+    public function getTimeShow()
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 114</br>Variable "time_show" is not in valid camel caps format

File: app/Http/Controllers/Backend/ListShowFilmController.php [hunk 16]
```diff
@@ -0,0 +1,134 @@
...
+
+    /**
+     * @param $id
+     * @return \Illuminate\Http\RedirectResponse
+     */
+
+    public function deleteTimeShow($id)
+    {
+        $model = $this->model->findOrFail($id);
+        if ($model->delete()) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 130</br>Expected 1 space after closing brace; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/ListShowFilmController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/ListShowFilmController.php</br>**Line**: 130</br>Expected 1 space(s) after ELSE keyword; 0 found

File: resources/views/layout/backend/listshowform.blade.php [hunk 17]
```diff
@@ -0,0 +1,63 @@
...
+
+@section('content')
+    <div class="form-category">
+        <form id="frm-category" name="frm-category" class="frm-category" method="post" enctype="multipart/form-data">
+            <input hidden="hidden" name="id" value="{{ isset($timeshow['id']) ? $timeshow['id'] : '' }}">
+            <div class="form-group">
+                <label for="name">{{ __('Status') }}:</label>
+                <select class="form-control" required="required" name="status">
+                    <option value="" hidden>{{ __('Select') }}</option>
+                    @if (Config('status'))
```
Comment (BunnyPi04) [inline resources/views/layout/backend/listshowform.blade.php]: config viết thường nhé

**Pull request merged**